lib/src/react16_suggestors/react_dom_render_migrator.dart

```diff
@@ -36,7 +36,8 @@ class ReactDomRenderMigrator extends GeneralizingAstVisitor
 //    }) as MethodDeclaration)?.name;
 
     if (node.methodName.name != 'render' ||
-        !const ['react_dom', 'reactDom'].contains(node.realTarget?.toSource?.call()) ||
+        !const ['react_dom', 'reactDom']
```

Comment:
I know this wasn't added in this PR but I have a similar concern with these as i do with the `react.component` stuff in the new codemod, are we just hoping that people named things in this convention? is there any way to like tell where the import came from? @greglittlefield-wf

Reply:
> are we just hoping that people named things in this convention
Yeah, that was the idea. And hopefully we'd catch any deviations during one time testing.

> is there any way to like tell where the import came from?
Yeah, I think we could look up the imports in the file and find the URI that corresponds to this prefix, might not be too bad. I could go either way on doing that or leaving it as-is.

Reply:
If you feel confident about it then im good with it as is, i just have no faith in humanity following a convention that is easily broken.

Reply:
🤷‍♂ Could do a search in Wdesk for `.render(` to see what the usages are like.


We could also make this a little more robust if we decide to stick with it
```dart
final normalizedPrefix = (node.realTarget?.toSource() ?? '').toLowerCase().replaceAll('_', '');
if (['reactdom', 'react'].contains(normalizedPrefix) ...)
```

Reply:
+3000